### PR TITLE
fix(containers/layout): pass children

### PR DESCRIPTION
**Pull request by jmfrancois**
**Please check if the PR fulfills these requirements**

- [x] The commit(s) message(s) follows our
  [guidelines](https://github.com/talend/tools/blob/master/tools-root-github/CONTRIBUTING.md#commit-message-format)
- [x] Tests for the changes have been added (for bug fixes / features)
- [ ] Docs have been added / updated (for bug fixes / features)

**What is the current behavior?** (You can also link to an open issue here)

The HomeLIstView doesn't pass children and has no unit tests.
We mock react-talend-components in containers tests but they display empty stuff and don't show props.
It's used in TFD.

**What is the new behavior?**

The HomeLIstView pass children props and has unit tests.
The mock of react-talend-components display props (so many snapshots has changed)

screen shot 2017-02-14 at 10 49 24

### CODE REVIEW

**Comment by jmfrancois** [comment]
:octocat: [Demo is available here](http://151.talend.surge.sh)

File: packages/containers/src/HomeListView/HomeListView.test.js [hunk 1]
```diff
@@ -10,10 +10,90 @@ import Connected, {
...
+			{ id: 'id', name: 'Id' },
+			{ id: 'label', name: 'Name' },
+		],
+	},
+	filter: {
+		placeholder: 'find an object',
+	},
+};
+
+const items = Immutable.fromJS([
```
Comment (acateland) [inline packages/containers/src/HomeListView/HomeListView.test.js]: fromJS can be directly imported 
```javascript
import Immutable, { Map } from 'immutable';
```
and used
```javascript
const items = fromJS([
```

File: packages/containers/src/HomeListView/HomeListView.test.js [hunk 2]
```diff
@@ -10,10 +10,90 @@ import Connected, {
...
+	],
+	titleProps: {
+		key: 'label',
+	},
+};
+
+const actions = {
+	//title: 'object:view',
+	//left: ['object:add'],
+	//items: ['object:delete'],
```
Comment (talend-glorieux) [inline packages/containers/src/HomeListView/HomeListView.test.js]: Please remove commented code.

**Comment by jmfrancois** [comment]
:octocat: [Demo is available here](http://151.talend.surge.sh)

**Comment by jmfrancois** [comment]
:octocat: [Demo is available here](http://151.talend.surge.sh)

**Comment by jmfrancois** [comment]
:octocat: [Demo is available here](http://151.talend.surge.sh)

**Pull request merged**